core/commands/dag/dag.go

```diff
@@ -241,3 +248,103 @@ var DagResolveCmd = &cmds.Command{
 	},
 	Type: ResolveOutput{},
 }
+
+var DagExportCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Streams the selected DAG as a .car stream on stdout.",
+		ShortDescription: `
+'ipfs dag export' fetches a dag and streams it out as a well-formed .car file.
+Note that at present only single root selections / .car files are supported.
+The output of blocks happens in strict DAG-traversal, first-seen, order.
+`,
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("root", true, false, "Expression evaluting to a single root of a dag to export").EnableStdin(),
```

Comment:
For now, let's just say "CID". Otherwise, the user isn't going to have any idea what to pass in. We can change the help text later when we add support for additional selectors.

Reply:
Fixed